cargo-dist/src/errors.rs

```diff
@@ -555,6 +555,20 @@ pub enum DistError {
     #[error("No bundle identifier was specified")]
     #[diagnostic(help("Please either enter a bundle identifier, or disable the Mac .pkg"))]
     MacPkgBundleIdentifierMissing {},
+
+    /// Project depends on a too-old axoupdater
+    #[error("Your project ({package_name}) uses axoupdater as a library, but the version specified ({your_version}) is older than the minimum supported version ({minimum}),")]
```

Comment:
it'd be great if we linked folks to a page in the docs that explained this a bit more

Reply:
Good call, I'll write a docs page for this and link to it.